src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1203,7 +1210,7 @@ void OrbitMainWindow::OnProcessListUpdated(
     target_label_->setToolTip({});
     target_process_state_ = TargetProcessState::kRunning;
   }
-  UpdateCaptureStateDependentWidgets();
+  UpdateProcessConnectionStateDependentWidgets();
```

Comment:
If you remove `UpdateCaptureStateDependentWidgets`, you need to duplicate the logic around `assume_target_process_is_running` into `UpdateProcessConnectionStateDependentWidgets` - the Start-Button should be disabled when Orbit is not connected to a process, or when the connection got lost.

Reply:
On main, the variable is now called `is_target_process_running`, btw, I didn't have Anton's latest changes.

Reply:
Added the logic around `is_target_process_running` into `UpdateProcessConnectionStateDependentWidgets`. Thanks for the comments. Sorry that I didn't realize this. I will be more careful the next time : )